control-plane/p4RuntimeArchStandard.h

```diff
@@ -479,6 +523,84 @@ struct Digest {
                                         // declaration.
 };
 
+//// The information about a emvlt extern, needed for serialization
+struct param_t {
+    uint32_t id;
+    cstring name;
+    uint32_t bitwidth;
+};
+struct MatchValueLookupTable {
+    const cstring name;
+    uint32_t key_bitwidth;
+    std::vector<param_t> params;
+    const uint32_t size;
+    const IR::IAnnotated* annotations;
+    MatchValueLookupTable(cstring n, uint32_t kbw, std::vector<param_t> params_list,
+                          uint32_t sz, const IR::IAnnotated* annos = nullptr) :
+        name(n), key_bitwidth(kbw), params(params_list), size(sz), annotations(annos) {}
+    static void add_mvlut_param(uint32_t &param_count, std::vector<param_t>* params_list,
```

Comment:
I think we should add a new lines between the variable declarations, constructor call and the last static func